src/app/api/(auth)/logout/route.ts

```diff
@@ -0,0 +1,36 @@
+import { cookies } from 'next/headers';
+import { NextResponse } from 'next/server';
+import { verify } from 'jsonwebtoken';
+
+export async function GET(request: Request) {
+	const cookie = cookies().get('jwt');
+
+	if (!cookie)
+		return NextResponse.json(
+			{ msg: 'Autentiserings-id saknas' },
```

Comment:
This will not be rendered on the screen right? In that case maybe use english instead. There is a couple of places in the code that is the same thing.

Reply:
Yes, you are right. Fixed ✅